### ISSUE TITLE
Support for lambda @ edge

### ISSUE THREAD

**Issue by bpavlov** [claygregory/serverless-prune-plugin#10]
Hello guy,

I received that kind of error:
Lambda was unable to delete LAMBADAQUALIFIEDARN:2 because it is a replicated function. Please see our documentation for Deleting Lambda@Edge Functions and Replicas.

Is there any way do delete lambda @ edge non-used version(s) or some way to prevent plugin from deleting only lambda @ edge versions?

**Comment by claygregory** [comment]
Hi @bpavlov,

Just pushed out a fix for this in v1.3.2 — support was previously implemented in #5. The check is built over the error message returned from AWS, but seems they've added additional information to their message recently, so updated error filter to reflect this.

Thanks for the report!

**Comment by EvgeniyGorbachev** [comment]
Hi, @claygregory.
I have the same error like above with plugin version 1.4.1:
```
Serverless Error ---------------------------------------
 
  Lambda was unable to delete arn:aws:lambda:us-east-1:11111:function:xxxxxxx:6 because it is a replicated function. Please see our documentation for Deleting Lambda@Edge Functions and Replicas.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              10.15.3
     Framework Version:         1.63.0
     Plugin Version:            3.3.0
     SDK Version:               2.3.0
     Components Core Version:   1.1.2
     Components CLI Version:    1.4.0
```
